tdvt/tdvt/tdvt.py

```diff
@@ -417,7 +417,9 @@ def get_level_of_parallelization(args):
     '''
 run_connectors_test_usage_text = '''
     The commands below can be used to run the connectors tests.
-    'filename.xml' in the below commands is the path of the xml test file used to run the tests.
+    'filename.txt' in the below commands is the path of the xml test file used to run the tests.
```

Comment:
Was there a reason this went back to `filename.txt` rather than `.xml`?

Reply:
Oops, git mis match. updating it